module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,17 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <array>
+
+void removeVowels(std::vector<std::string>& vec) {
+    std::array<char, 12>
+        vowels{'a', 'e', 'i', 'o', 'u', 'y', 'A', 'E', 'I', 'O', 'U', 'Y'};
```

Comment:
You can make it const or constexpr and I would consider moving this to the header.

Reply:
not a scope of the task, corrected on other branch